pg-admin/servers.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Servers": {
+    "1": {
+      "Name": "transcendence",
+      "Group": "Servers",
+      "Host": "db",
+      "Port": 5432,
+      "MaintenanceDB": "postgres",
+      "Username": "postgres",
+      "PassFile": "/pgpass",
+      "SSLMode": "prefer",
+      "SSLCompression": 0,
+      "Timeout": 10,
+      "UseSSHTunnel": 0,
+      "TunnelPort": "22",
+      "TunnelAuthentication": 0
+    }
+  }
+}
```

Comment:
The postgresql configuration so when you log in server is already loaded and no need to configure IPs, users or passwords